README.adoc

```diff
@@ -159,3 +159,34 @@ defer c.Close()
 // new message.
 err = c.Subscribe("destination name", callback)
 ----
+
+=== Running Tests and Benchmarks
+
+You can use the `make` command to run tests and benchmarks on your local computer.
+
+To run the tests, use the `make test` command.
+
+==== Run unit tests:
+
+[source]
+----
+$ make test
+----
+
+Benchmarks should be run using an external STOMP broker.
+
+Before running the Benchmarks, start a STOMP broker, for example ArtimisMQ:
```

Comment:
The name of the broker is _ActiveMQ Artemis_.

Reply:
:+1: